src/sections/navbar.html

```diff
@@ -0,0 +1,38 @@
+<div class="bg-PrimaryColor flex justify-between p-4 fixed w-full top-0 z-10">
```

Comment:
you can increase the padding (pt or py) because the vertical space above the nav bar from the figma page is larger than the one we have here.